src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,99 @@
+.container {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+  max-width: 200px;
+}
+
+.picture {
+  height: 134px;
+  width: 160px;
+  margin-top: 16px;
+}
+
+.product-name {
```

Comment:
Consider using consistent naming convention for class names. `product-name` should be `productName`. 🤔